public/js/confirmation.js

```diff
@@ -1,23 +1,25 @@
 //Afficher le prix de la commande avec le numéro de suivi(order)
-"use strict"; // Une variante plus restreinte de JavaScript (par exemple, on ne peut pas utiliser de variable avant de l'avoir définie). Elle permet d'obtenir de meilleures performances et de faciliter le débogage.
-
-
-
-
-
-
-
-function ordernumber(){
-
-    const returnBackEnd = JSON.parse(localStorage.getItem("order"));
-    console.log(returnBackEnd)
-    const prixTotal = localStorage.getItem("Prixtotal");
-
-    const confirmation = document.createElement("p");
-    confirmation.classList.add("p-5","fs-5");
-    confirmation.textContent = "Nous validons M. ou Mme " + returnBackEnd.contact.lastName + " votre commande de " + prixTotal + "€. Votre numéro de facture est le " + returnBackEnd.orderId + ". Nous vous remercions de votre achat.";
-
-    document.getElementById("order").appendChild(confirmation);
+"use strict"; // Une variante plus restreinte de JavaScript (par exemple, on ne peut pas utiliser de variable avant de l"avoir définie). Elle permet d"obtenir de meilleures performances et de faciliter le débogage.
+
+
+//fonction permettant d"afficher le numéro de facture
+function displayOrderNumber(){
+
+    const returnBackEnd = JSON.parse(localStorage.getItem("order")); //Je crée une variable contenant le localStorage de la clé "order" sous forme d"objet
+    console.log(returnBackEnd);
+    const prixTotal = localStorage.getItem("Prixtotal"); //Je crée une variable contenant le prixtotal stocké dans la page panier
+
+    const confirmation = document.getElementById("order").innerHTML =
+        (
+            '<p class="p-5 fs-5">Nous validons M. ou Mme <b>'
+            + returnBackEnd.contact.lastName 
+            +'</b> votre commande de <b>'
+            + prixTotal
+            +' €</b>.<br>Votre numéro de facture est le <b>'
+            + returnBackEnd.orderId
+            +'</b>.<br>Nous vous remercions de votre achat.</p>');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/JulienBrazzalotto/JulienBrazzalotto_5_240921/pullRequest?prid=8291925)
```suggestion
            +"</b>.<br> Nous vous remercions de votre achat.</p>");
```